composer/trainer/trainer.py

```diff
@@ -1106,6 +1106,11 @@ def __init__(
         self._checkpoint_saver = None
         latest_remote_file_name = None
         if save_folder is not None:
+            if save_weights_only:
+                warnings.warn(
```

Comment:
Imo fine to remove? In general warnings.warn means a user has an action they need to do.

Maybe a log.info saying as of version 0.16, save_weights_only=True includes metadata and integrations for easier export to inference. This may break workflows which rely on only one key being present in checkpoint.

Reply:
Yeah I'd go with info since the user has no way to get rid of this warning.